utils/remote.cc

```diff
@@ -3167,9 +3166,9 @@ static int processArgs(char const* rpcurl, int argc, char const* const* argv, Co
                 {
                     auto top = tr_variant{};
                     tr_variantInitDict(&top, 2);
-                    tr_variantDictAddStr(&top, TR_KEY_method, "rename");
+                    tr_variantDictAddStr(&top, TR_KEY_method, "torrent-rename-path"sv);
                     auto* args = tr_variantDictAddDict(&top, Arguments, 3);
-                    tr_variantDictAddStr(args, TR_KEY_path, rename_from);
+                    tr_variantDictAddStr(args, TR_KEY_path, config.rename_from);
                     tr_variantDictAddStr(args, TR_KEY_name, optarg);
                     addIdArg(args, config);
                     status |= flush(rpcurl, &top, config);
```

Comment:
If you want to improve this further, I'd say we need to clear out `rename_from` after flushing, so that users are required to pass `--path` again instead of us trying to rename the same path twice (in case `--rename` is passed more than once) which makes no sense.